spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -55,7 +55,12 @@
 
   context 'confirmed user is signed in and tries to confirm again' do
     it 'redirects the user to the profile' do
-      sign_up_and_2fa
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
Yes I can see your point. I think I also felt like, in some specs at least, what we were testing was the flow itself, so having the steps of the flow obscured was actually obscuring the spec exercise.

I think that if we *do* want to use this, we should name it more explicitly and then use it in more places. Now that this flow is only the case for LOA1 / non-SP situations, having it named `sign_up_and_2fa_user` and located in a different file is confusing because we only want the "click recovery code" part to happen for non-LOA3.